ftl/README.md

```diff
@@ -28,3 +49,7 @@ python ftl/ftl_node_integration_tests_yaml.py | gcloud container builds submit -
 python ftl/ftl_php_integration_tests_yaml.py | gcloud container builds submit --config /dev/fd/0 .
 gcloud container builds submit --config ftl/ftl_python_integration_tests.yaml .
 ```
+
+## FTL Runtime Design Documents
+[php](https://docs.google.com/document/d/1AB255g8N-J7IYEbhmiTRf29Ox1afEgs3df8GK4NBhrk/edit?usp=sharing)
```

Comment:
These links probably won't work publicly. Consider leaving them out until you port them to markdown.

Reply:
I made this links to work publicly from my other gmail account.  These documents are community docs w/ internal Google references removed